Purchases/Identity/RCPurchaserInfoManager.m

```diff
@@ -181,6 +181,7 @@ - (void)purchaserInfoWithAppUserID:(NSString *)appUserID
 }
 
 - (void)clearPurchaserInfoCacheForAppUserID:(NSString *)appUserID {
+    [RCLog debug:[NSString stringWithFormat:@"%@", RCStrings.purchaserInfo.invalidating_purchaserinfo_cache]];
```

Comment:
This one is being migrated now (I'm working on RCPurchaserInfoManger), so feel free to keep it or remove it.